control_panel_api/management/commands/add_oidc_statement_to_users_roles.py

```diff
@@ -0,0 +1,65 @@
+import json
+import logging
+
+import boto3
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from control_panel_api.models import User
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Adds OIDC statement to all users roles trust policies"
+
+    def handle(self, *args, **options):
+        iam = boto3.resource('iam')
+        for user in User.objects.all():
+            # Read role trust policy
+            iam_role = iam.Role(user.iam_role_name)
+            assume_role_policy_document = iam_role.assume_role_policy_document
+
+            if not self._has_oidc_statement(assume_role_policy_document):
+                # Add OIDC statement
+                oidc_statement = self._oidc_statement(user.auth0_id)
+                assume_role_policy_document['Statement'].append(oidc_statement)
+
+                # Update role trust policy
+                assume_role_policy = iam_role.AssumeRolePolicy()
+                assume_role_policy.update(
+                    PolicyDocument=json.dumps(assume_role_policy_document)
+                )
+                logger.info(f'OIDC statement added to "{user.iam_role_name}" trust policy.')
+            else:
+                logger.info(f'OIDC statement already found in "{user.iam_role_name}" trust policy. Skipping.')
```

Comment:
Is this log output necessary? It seems not, unless it's a noteworthy fact that the OIDC statement is already found.